include/cantera/thermo/DebyeHuckel.h

```diff
@@ -33,7 +33,7 @@ namespace Cantera
 #define DHFORM_PITZER_BETAIJ 4
 //! @}
 /*!
- *  @name  Acceptable ways to calculate the value of A_Debye
+ * @name  Acceptable ways to calculate the value of A_Debye
```

Comment:
Why isn't this one being converted to a `//!` comment?

Reply:
This is a module grouping, which I didn't touch yet (I focused on member groupings). I can though - let me know ...

Reply:
Ah, I see. I only commented because this line was in the diff.